qt/scientific_interfaces/Indirect/IndirectSymmetriseView.cpp

```diff
@@ -0,0 +1,434 @@
+  m_propTrees["SymmPropTree"]->setFactoryForManager(m_dblManager, doubleEditorFactory);
+
+  // Raw Properties
+  m_properties["EMin"] = m_dblManager->addProperty("EMin");
+  m_dblManager->setDecimals(m_properties["EMin"], numDecimals);
+  m_propTrees["SymmPropTree"]->addProperty(m_properties["EMin"]);
+  m_properties["EMax"] = m_dblManager->addProperty("EMax");
+  m_dblManager->setDecimals(m_properties["EMax"], numDecimals);
+  m_propTrees["SymmPropTree"]->addProperty(m_properties["EMax"]);
+
+  QtProperty *rawPlotProps = m_grpManager->addProperty("Raw Plot");
+  m_propTrees["SymmPropTree"]->addProperty(rawPlotProps);
+
+  m_properties["PreviewSpec"] = m_dblManager->addProperty("Spectrum No");
+  m_dblManager->setDecimals(m_properties["PreviewSpec"], 0);
+  rawPlotProps->addSubProperty(m_properties["PreviewSpec"]);
+
+  // Preview Properties
+  // Mainly used for display rather than getting user input
+  m_properties["NegativeYValue"] = m_dblManager->addProperty("Negative Y");
+  m_dblManager->setDecimals(m_properties["NegativeYValue"], numDecimals);
+  m_propTrees["SymmPVPropTree"]->addProperty(m_properties["NegativeYValue"]);
+
+  m_properties["PositiveYValue"] = m_dblManager->addProperty("Positive Y");
+  m_dblManager->setDecimals(m_properties["PositiveYValue"], numDecimals);
+  m_propTrees["SymmPVPropTree"]->addProperty(m_properties["PositiveYValue"]);
+
+  m_properties["DeltaY"] = m_dblManager->addProperty("Delta Y");
+  m_dblManager->setDecimals(m_properties["DeltaY"], numDecimals);
+  m_propTrees["SymmPVPropTree"]->addProperty(m_properties["DeltaY"]);
+
+  auto const xLimits = m_uiForm.ppRawPlot->getAxisRange(AxisID::XBottom);
+  auto const yLimits = m_uiForm.ppRawPlot->getAxisRange(AxisID::YLeft);
+
+  // Indicators for Y value at each EMin position
+  auto negativeEMinYPos =
+      m_uiForm.ppRawPlot->addSingleSelector("NegativeEMinYPos", MantidWidgets::SingleSelector::YSINGLE, 0.0);
+  negativeEMinYPos->setColour(Qt::blue);
+  negativeEMinYPos->setBounds(std::get<0>(yLimits), std::get<1>(yLimits));
+
+  auto positiveEMinYPos =
+      m_uiForm.ppRawPlot->addSingleSelector("PositiveEMinYPos", MantidWidgets::SingleSelector::YSINGLE, 1.0);
+  positiveEMinYPos->setColour(Qt::red);
+  positiveEMinYPos->setBounds(std::get<0>(yLimits), std::get<1>(yLimits));
+
+  // Indicator for centre of symmetry (x=0)
+  auto centreMarkRaw = m_uiForm.ppRawPlot->addSingleSelector("CentreMark", MantidWidgets::SingleSelector::XSINGLE, 0.0);
+  centreMarkRaw->setColour(Qt::cyan);
+  centreMarkRaw->setBounds(std::get<0>(xLimits), std::get<1>(xLimits));
+
+  // Indicators for negative and positive X range values on X axis
+  // The user can use these to move the X range
+  // Note that the max and min of the negative range selector corespond to the
+  // opposite X value
+  // i.e. RS min is X max
+
+  auto positiveERaw = m_uiForm.ppRawPlot->addRangeSelector("PositiveE");
+  positiveERaw->setColour(Qt::darkGreen);
```

Comment:
can we avoid the colour green? You have used red and I think we should be mindful of red-green colour blindness